spdmlib/src/common/session.rs

```diff
@@ -110,6 +110,20 @@ pub struct SpdmSessionRuntimeInfo {
     pub digest_context_l1l2: Option<SpdmHashCtx>,
 }
 
+// for session backup-recovery purpose
+#[derive(Default, Debug, Clone)]
+pub struct CriticalDheSessionInfo {
```

Comment:
1. why call `critical`? it is just backup.
2. why call `dhe`? PSK should also be applicable.